docs/usage/tables/parquet.md

```diff
@@ -0,0 +1,120 @@
+├────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┤
+│marksheet...│unity           │default         │EXTERNAL        │DELTA           │[{"name":"id"...│file:///tmp/m...│Uniform table   │{"key1":"valu...│1721238005611   │1721238005611   │9a73eb46-adf0...│
+├────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┼────────────────┤
+│marksheet   │unity           │default         │MANAGED         │DELTA           │[{"name":"id"...│file:///Users...│Managed table   │{"key1":"valu...│1721238005595   │1721238005595   │c389adfa-5c8f...│
+└────────────┴────────────────┴────────────────┴────────────────┴────────────────┴────────────────┴────────────────┴────────────────┴────────────────┴────────────────┴────────────────┴────────────────┘
+```
+
+As you can see, there are 4 tables in this catalog. All 4 pre-loaded tables are in the DELTA format.
+
+Let's take a look at how we can create Parquet tables in Unity Catalog.
+
+## Create a Parquet table
+
+Use the `bin/uc table create ...` command with the `--format PARQUET` flag to create a new Parquet table in your Unity Catalog.
+
+This command has multiple parameters:
+
+- `full_name`: The full name of the table, which is a concatenation of the catalog name, schema name, and table name separated by dots (e.g., catalog_name.schema_name.table_name).
+- `columns`: The columns of the table in SQL-like format "column_name column_data_type". Supported data types include BOOLEAN, BYTE, SHORT, INT, LONG, FLOAT, DOUBLE, DATE, TIMESTAMP, TIMESTAMP_NTZ, STRING, BINARY, DECIMAL. Separate multiple columns with a comma (e.g., "id INT, name STRING").
+- `format`: [Optional] The format of the data source. Supported values are DELTA, PARQUET, ORC, JSON, CSV, AVRO, and TEXT. If not specified the default format is DELTA.
+- `storage_location`: The storage location associated with the table. It is a mandatory field for EXTERNAL tables.
+- `properties`: [Optional] The properties of the entity in JSON format (e.g., '{"key1": "value1", "key2": "value2"}'). Make sure to either escape the double quotes(\") inside the properties string or just use single quotes('') around the same.
+
+Run the command below with the correct `path/to/storage` to create a new PARQUET table with 2 colummns: `some_numbers` and `some_letters`.
+You can get the correct storage location using a `bin/uc table get ...` call to fetch some table metadata.
+
+```sh
+bin/uc table create --full_name unity.default.test --columns "some_numbers INT, some_letters STRING" --storage_location file:///Users/avriiil/Documents/git/my-forks/unitycatalog/etc/data/external/unity/default/tables/test
```

Comment:
Can you move this command earlier - say - line 45?  This way users can see the command right away?